packages/jellyfish-api-core/__tests__/category/account/accountHistoryCount.test.ts

```diff
@@ -121,12 +122,31 @@ describe('Account', () => {
       }
       const options2: AccountHistoryCountOptions = {
         txtype: DfTxType.POOL_SWAP
-
       }
       const count1 = await client.account.historyCount('mine', options1)
       const count2 = await client.account.historyCount('mine', options2)
 
       expect(count1 === count2).toStrictEqual(false)
     })
   })
+
+  it('should get accountHistoryCount for multiple txtypes at once', async () => {
+    const mintCount = await client.account.historyCount('mine', { txtype: DfTxType.MINT_TOKEN })
+    const poolSwapCount = await client.account.historyCount('mine', { txtype: DfTxType.POOL_SWAP })
+
+    await waitForExpect(async () => {
```

Comment:
Why do we need to use `waitForExpect` for this test?

Reply:
Yeah, it is unncessory. Removed here [`fe75304`](https://github.com/JellyfishSDK/jellyfish/pull/1943/commits/fe753048621b58f14c4ed9e5baba375faeb77838) tho, many other tests are also using it unnecessarily but did not bother removing those.